pkg/acquisition/file_reader.go

```diff
@@ -156,7 +156,9 @@ func InitReaderFromFileCtx(files []FileCtx) (*FileAcquisCtx, error) {
 				log.Errorf("error while globing '%s' : %v", fglob, err)
 				return nil, err
 			}
-
+			if len(files) == 0 {
+				return nil, fmt.Errorf("unable to glob path '%s'", fglob)
```

Comment:
if I'm not mistaken, it means that if we have a `acquis.yaml` with at least one glob that doesn't resolve to a file, it's going to prevent crowdsec from starting  :

```bash
$ cat config/acquis.yaml 
filenames:
 - /tmp/test*.log
 - /tmp/noppy.log
labels:
  type: syslog
               $ ./crowdsec -c dev.yaml 
...
              INFO[27-07-2020 11:31:53] Opening file '/tmp/test.log' (pattern:/tmp/test*.log) 
INFO[27-07-2020 11:31:53] Opening file '/tmp/testasd.log' (pattern:/tmp/test*.log) 
FATA[27-07-2020 11:31:53] Error while loading acquisition config : Failed to start acquisition : unable to start file acquisition, bailout unable to glob path '/tmp/noppy.log' 
                                                                                                                                                                          
```

I think a warning or even an error message is fair, but this is a bit too strict.

Reply:
fixed